### PR TITLE
db/context-detection

### PR DESCRIPTION
**Pull request by karolkolodziej**
Fixed content detection after adding a component.

### CODE REVIEW

File: test/cypress/integration/Dashboard/component-add.cy.js [hunk 1]
```diff
@@ -75,7 +77,7 @@ describe('Add component through UI', () => {
             );
             assert.closeTo(
                 component.dimensions.width,
-                270,
```
Comment (pawellysy) [inline test/cypress/integration/Dashboard/component-add.cy.js:78]: Fails on circleCI. Could you investigate, or find a better way of calculating the element's width?

**Review by pawellysy — CHANGES_REQUESTED** [review]
Changes look good, minor comment to cypress test.

File: test/cypress/integration/Dashboard/component-add.cy.js (hunk 1 above)
Comment (karolkolodziej) [reply test/cypress/integration/Dashboard/component-add.cy.js:78]: I tried different ways of calculating that but non were ideal. 
Decided to simply decrease sensitivity for now.

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

**Review by sebastianbochan — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]
LGTM!

**Pull request merged**